accounts/models/connections.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from Gp_Backend.s3_storages import MediaStorage
+
+class connection(models.Model):
+    account = models.OneToOneField("accounts.Account", on_delete=models.CASCADE, related_name='connection')
+    name = models.CharField(max_length=100, null= False)
+    image = models.ImageField(upload_to='connection/', storage=MediaStorage, null=True, blank=True)
+    age = models.IntegerField(max_length=50, null =False)
+    relation = models.CharField(max_length=100, null=False)
+    address = models.CharField(max_length=200, null= True, blank = True)
+    biograghy = models.CharField(max_length=200, null=True, blank = True) 
```

Comment:
- following the pep8 standards when we define parameters of a function or class we do the following format `(key=value, key=value, key=value, key=value, ...)`
- I  mean there are no spaces between the `ley` and `=` and `value` and `,` but there's a space between every `,` and the next parameter.